libs/lib-client/core/src/services/feature-toggles.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable, APP_INITIALIZER, FactoryProvider } from '@angular/core';
+import { tap, catchError } from 'rxjs/operators';
+import { EMPTY } from 'rxjs';
+import { RestApiService } from './restapi';
+
+@Injectable({ providedIn: 'root' })
+export class FeatureToggleService {
+  features: { [featureName: string]: any };
+
+  isFeatureEnabled(featureName: string) {
+    return this.features && !!this.features[featureName];
```

Comment:
When I run the following command in windows to set the process variables
```
cmd /v /c "set FLOGO_FEATURES_ANAND=true&& set FLOGO_FEATURES_PANDURANGI_X=true&& yarn start server"
```

I get the following result (http://localhost:3303/api/v2/_/features):
```
{"ANAND":"true","PANDURANGI_X":"true"}
```

If you notice, the value is not boolean type, so the response will always be true. Also it is failing the concept of 
```
# disable
# FLOGO_FEATURES_FLAG_X=
```

Reply:
That's how env variables usually work, they're always strings, I don't want to get too fancy with this implementation and introduce parsing of variables since this is mostly for development only.

>If you notice, the value is not boolean type, so the response will always be true. ?>Also it is failing the concept of
>
> ```
> # disable
> # FLOGO_FEATURES_FLAG_X=
> ```

I didn't understand this part.

Reply:
I've updated the .env file comments a bit to clarify the flags behavior :v:

Reply:
Okay the other comment clarifies that you purposely left the value empty. The comment in `env.example` files gives me an impression that you are proposing a X as an suffix to know that it is disabled. :)